src/main/java/net/consensys/wittgenstein/protocols/GSFSignature.java

```diff
@@ -656,8 +656,8 @@ private static GSFSignature newProtocol() {
     double deadR = 0.10;
     double tsR = .85;
 
-    String nb = RegistryNodeBuilders.name(true, false, 0.33);
-    String nl = NetworkLatency.AwsRegionNetworkLatency.class.getSimpleName();
+    String nb = RegistryNodeBuilders.ALL_CITIES;// name(true, false, 0.33);
```

Comment:
Why this change?

Reply:
the
> RegistryNodeBuilders.name

function currently doesn't support (All_CITIES, Real Latency) option. We can update it as a separate PR.